labs/lab3/chapter3.md

```diff
@@ -64,8 +64,6 @@ $ sudo podman stop $(sudo podman ps -ql)
 $ sudo podman rm $(sudo podman ps -ql)
 ```
 
-If we are confident in what we are doing we can also "single-line" the above with `sudo podman rm -f $(sudo podman ps -ql)` by itself.
```

Comment:
why did you kill this?

Reply:
i had removed this before i moved the registry to the end of this chapter. the reason being, if the prior stop/rm command had already been executed, then running that command was actually deleting the registry... which is needed next in this chapter and for the remaining chapters. my intent was to avoid confusion at summit.

that said, now that we don't have them start the registry until after this step... i need to add this back.

Reply:
restored